### ISSUE TITLE
Passing an object to toolbarButtons throws an exception

### ISSUE THREAD

**Issue by johnnyoshika** [froala/wysiwyg-editor#4281]
##### Expected behavior.

Per the documentation here: https://froala.com/wysiwyg-editor/docs/options/#toolbarButtons, we should be able to pass an object to `toolbarButtons`. For example:

```
      new FroalaEditor("#edit", {
        toolbarButtons: 	{
          'moreText': {
            'buttons': ['bold', 'italic', 'underline', 'strikeThrough', 'subscript', 'superscript']
          },
        }
      })
```

##### Actual behavior.

Passing an object to `toolbarButtons` throws an exception at this line of froala_editor.js:

```
if (editor.opts.toolbarButtons && editor.opts.toolbarButtons.indexOf('trackChanges') > -1) {
```

The problem occurs because `indexOf` is being called on `toolbarButtons`, and such a method doesn't exist.

Even with this exception,  the toolbars will still load. However, clicking on any of the buttons on the toolbar won't do anything because of a script error as can be seen from this demo:

https://jsfiddle.net/johnny_examind/59gkohz0/9/

##### Steps to reproduce the problem.

Load Froala with this:

```
      new FroalaEditor("#edit", {
        toolbarButtons: 	{
          'moreText': {
            'buttons': ['bold', 'italic', 'underline', 'strikeThrough', 'subscript', 'superscript']
          },
        }
      })
```

Try clicking one of the toolbar button.

##### Editor version.

4.0.1

##### OS.

Windows 10

##### Browser.

Chrome Version 91.0.4472.124

**Comment by acwr** [comment]
Same problem stopping us from upgrading to v4. Looks like a dup of this:
https://github.com/froala/wysiwyg-editor/issues/4262

**Comment by johnnyoshika** [comment]
Thanks @acwr, it does look like a duplicate.

**Comment by johnnyoshika** [comment]
This has been fixed in version 4.0.4
